src/test/test-client.ts

```diff
@@ -0,0 +1,53 @@
+/// <reference path="../../typings/knockout/knockout.d.ts" />
+/// <reference path="../../typings/jasmine/jasmine.d.ts" />
+/// <reference path="../../typings/socket.io-client/socket.io-client.d.ts" />
+
+import { App } from "../common/client-app";
+import * as viewModel from "../common/view-model";
+import * as model from "../common/model";
+import * as ko from "knockout";
+import { BoardVM } from "../koclient/ko-view-model";
+
+
+describe("Test boad client", function(){
+  it("When a client recives a change on name, they are applied on the board", function(){
+    // Arrange
+    var board: model.Board = { name: "Old Name" };
+    var boardVM: BoardVM = new BoardVM();
+    boardVM.update(board);
+    var client: App = new App(boardVM);
+    var newName = "New Name";
+    var change: model.PatchMessage = {patch: [{op: "add", path: "/name", value: newName}]};
+
+    // Act
+    client.onMessage(change);
+    board = boardVM.toPlain();
```

Comment:
I think that the value returned by `boardVM.toPlain()`  it's more representative because the name attribute of BoardVM it's a knockout's observable type variable